src/EditorFeatures/CSharpTest/ChangeSignature/ChangeSignature_CheckAllSignatureChanges.cs

```diff
@@ -10,6 +10,21 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.ChangeSignature
 {
     public partial class ChangeSignatureTests : AbstractChangeSignatureTests
     {
+        [WorkItem(8333, "https://github.com/dotnet/roslyn/issues/8333")]
+        [WpfFact, Trait(Traits.Feature, Traits.Features.ChangeSignature)]
+        public async Task TestDoNotShowSignatureChangeForExpressionBody()
+        {
+            var markup = @"
+class Point
+{
+    int x; int y;
+    public Point(int x, int y) { }
+    public Point Move1(int dx, int dy) [|=>|] new Point(x + dx, y + dy);
```

Comment:
Ah, spoke a minute early. Never mind the above. It does seem to show up on both selection and/or caret position. So, I think this is good.